src/magnetic_spacegroup.c

```diff
@@ -182,9 +182,9 @@ MagneticDataset *msg_identify_magnetic_space_group_type(
 
     msgtype = msgdb_get_magnetic_spacegroup_type(uni_number);
     if (msgtype.type != type) {
-        warning_print("Inconsistent MSG type:\n");
-        warning_print("  From FSG and XSG: %d\n", type);
-        warning_print("  From DB matching: %d\n", msgtype.type);
+        info_print("Inconsistent MSG type:\n");
+        info_print("  From FSG and XSG: %d\n", type);
+        info_print("  From DB matching: %d\n", msgtype.type);
         goto err;
```

Comment:
This is an unreachable situation without something floating-point error. So, I gueess this will be info.

Reply:
If it's unreachable like an `assert` it should be `warning`. Assuming the floating point error is not a dynamic tolerance

Reply:
Since these are tied to a user input, these would not be classified as `assert` (assert is only for debug if you are sure that users would not encounter it, that is why it is gated by `NDEBUG`), but as runtime `try-except`-like functionality.

It seems it could be either `info` or `warning`, but these are quite hard to detangle, so I am not sure how to classify either, since some are in a for loop, some are not. Ultimately we are mostly limited by the C language that we cannot use libraries like `spdlog`, which would make these issues obsolete (since we can dynamically enable/disable/redirect logging, including having python native logs)

Reply:
Let's go with `info`.